### PR TITLE
Fluff Item: Decemviri

### PR DESCRIPTION
**Pull request by Aurorablade**
ckey:Decemviri
Player: Sylus Cain
Fluff Type: Space pod Mod kit

:cl: Fethas
rscadd:Adds Decemviris Fluff
/:cl:

### CODE REVIEW

**Comment by IK3I** [comment]
Me thinks someone played starfox when they were younger.

**Comment by Necaladun** [comment]
just way too fucking cool :+1: :100: :1st_place_medal:

File: code/modules/spacepods/spacepod.dm [hunk 1]
```diff
@@ -67,9 +67,13 @@
 
 	var/move_delay = 2
 	var/next_move = 0
+	var/can_paint = TRUE
 
 /obj/spacepod/proc/apply_paint(mob/user as mob)
 	var/part_type
+	if(can_paint == FALSE)
```
Comment (tigercat2000) [inline code/modules/spacepods/spacepod.dm]: !can_paint

File: code/modules/spacepods/spacepod.dm [hunk 2]
```diff
@@ -67,9 +67,13 @@
...
+		to_chat(user, "You can't repaint this type of pod!")
```
Comment (tigercat2000) [inline code/modules/spacepods/spacepod.dm]: Add `return` after this and span classes to this message

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -1262,3 +1262,29 @@
...
+	name = "Spacepod mod kit"
+	desc = "a kit on tools and a blueprint detailing how to reconfigure a spacepod"
+	icon_state = "modkit"
+	used = 0
+
+/obj/item/device/fluff/decemviri_spacepod_kit/afterattack(atom/target, mob/user, proximity)
+	if(!proximity || !ishuman(user) || user.incapacitated())
+		return
+	if(used)
+		to_chat(user, "<span class='warning'>The doesn't have any spare parts to use on another pod!</span>")
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: This entire block is pointless. The object qdels itself after being applied

File: code/modules/customitems/item_defines.dm [hunk 4]
```diff
@@ -1262,3 +1262,29 @@
...
+			var/obj/spacepod/pod = target
+			pod.icon = 'icons/48x48/custom_pod.dmi'
+			pod.icon_state = "pod_dece"
+			pod.name = "sleek spacepod"
+			pod.desc = "A modified varient of a space pod."
+			pod.can_paint = FALSE
+			used = 1
+			qdel(src)
+			return
+		to_chat(user, "<span class='warning'>You can't modify [target]!</span>")
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: Make this under `if(!istype(target, /obj/spacepod))`, add a return, and then move the rest of the code down

**Comment by taukausanake** [comment]
We need more space pod mod kits now

**Comment by Ty-Omaha** [comment]
That's hot

**Comment by Fox-McCloud** [comment]
please update.

**Comment by Aurorablade** [comment]
HOW DO I KEEP FORGETTING TO SYNC MY SHIT?!

**Comment by Decemviri1** [comment]
*stares longingly*

**Pull request merged**